app/controllers/admin/api/backend_apis_controller.rb

```diff
@@ -0,0 +1,148 @@
+  ##~ op.group = "backend_api"
+  #
+  ##~ @parameter_page = {:name => "page", :description => "Page in the paginated list. Defaults to 1.", :dataType => "int", :paramType => "query", :defaultValue => "1"}
+  ##~ @parameter_per_page = {:name => "per_page", :description => "Number of results per page. Default and max is 500.", :dataType => "int", :paramType => "query", :defaultValue => "500"}
+  ##
+  ##~ op.parameters.add @parameter_access_token
+  ##~ op.parameters.add @parameter_page
+  ##~ op.parameters.add @parameter_per_page
+  ##
+  #
+  def index
+    respond_with(current_account.backend_apis.oldest_first.paginate(pagination_params))
+  end
+
+  ##~ e = sapi.apis.add
+  ##~ e.path = "/admin/api/backend_apis.json"
+  ##~ e.responseClass = "backend_api"
+  #
+  ##~ op = e.operations.add
+  ##~ op.httpMethod = "POST"
+  ##~ op.summary    = "Backend API Create"
+  ##~ op.description = "Creates a Backend API."
+  ##~ op.group = "backend_api"
+  #
+  ##~ op.parameters.add @parameter_access_token
+  ##~ op.parameters.add @parameter_backend_api_name
+  ##~ op.parameters.add @parameter_backend_api_description
+  ##~ op.parameters.add @parameter_backend_api_private_endpoint
+  #
+  def create
+    backend_api = current_account.backend_apis.create(create_params)
+    respond_with(backend_api)
+  end
+
+  ##~ e = sapi.apis.add
+  ##~ e.path = "/admin/api/backend_apis/{id}.json"
+  ##~ e.responseClass = "backend_api"
+  #
+  ##~ op = e.operations.add
+  ##~ op.httpMethod = "GET"
+  ##~ op.summary = "Backend API Read"
+  ##~ op.description = "Returns a backend API."
+  ##~ op.group = "backend_api"
+  #
+  ##~ op.parameters.add @parameter_access_token
+  ##~ op.parameters.add @parameter_backend_api_id_by_id
+  #
+  def show
+    respond_with(backend_api)
+  end
+
+  ##~ e = sapi.apis.add
+  ##~ e.path = "/admin/api/backend_apis/{id}.json"
+  ##~ e.responseClass = "backend_api"
+  #
+  ##~ op            = e.operations.add
+  ##~ op.httpMethod = "PUT"
+  ##~ op.summary    = "Backend API Update"
+  ##~ op.description = "Updates a backend API."
+  ##~ op.group = "backend_api"
+  #
+  ##~ op.parameters.add @parameter_access_token
+  ##~ op.parameters.add @parameter_backend_api_id_by_id
+  ##~ op.parameters.add @parameter_backend_api_name
+  ##~ op.parameters.add @parameter_backend_api_description
+  ##~ op.parameters.add @parameter_backend_api_private_endpoint
+  #
+  def update
+    backend_api.update(update_params)
+    respond_with(backend_api)
+  end
+
+  ##~ e = sapi.apis.add
+  ##~ e.path = "/admin/api/backend_apis/{id}.json"
+  ##~ e.responseClass = "backend_api"
+  #
+  ##~ op            = e.operations.add
+  ##~ op.httpMethod = "DELETE"
+  ##~ op.summary    = "Backend API Delete"
+  ##~ op.description = "Deletes a backend API."
+  ##~ op.group = "backend_api"
+  #
+  ##~ op.parameters.add @parameter_access_token
+  ##~ op.parameters.add @parameter_backend_api_id_by_id
+  #
+  def destroy
+    backend_api.destroy
```

Comment:
Good for now though we should probably wait for this PR to be merged to implement it correctly.
Wdyt?
https://github.com/3scale/porta/pull/1144

Reply:
Right 👍 I will wait for that PR and then do it correctly here 😉 👍